README.md

```diff
@@ -92,6 +92,37 @@ See https://react-dropzone.netlify.com/#proptypes
 
 *Important*: `react-dropzone` doesn't manage dropped files. You need to destroy the object URL yourself whenever you don't need the `preview` by calling `window.URL.revokeObjectURL(file.preview);` to avoid memory leaks.
 
+### Testing
+
+*Important*: for performance reasons, `react-dropzone` makes its drag'n'drop callbacks asynchronous. In order to properly test this, you may want to utilize a helper function to run all promises like this:
```

Comment:
It's not performance reasons but plugins. See https://github.com/react-dropzone/react-dropzone/pull/616